docs/source/usage.rst

```diff
@@ -100,114 +100,68 @@ To make this changes possible you can always create your own
 
 .. code-block:: python
 
-    from prettyconf.configuration import Configuration
+    from prettyconf import Configuration
 
     config = Configuration()
 
 .. warning:: ``prettyconf`` will skip configuration files inside ``.zip``,
    ``.egg`` or wheel packages.
 
 
-Set the starting path
-+++++++++++++++++++++
+Customizing the configuration file location
++++++++++++++++++++++++++++++++++++++++++++
 
 By default library will use the directory of the file where ``config()`` was
-called as the start directory to look for configuration files. Consider the
-following file structure:
+called as the start directory to look for a ``.env`` configuration file.
+Consider the following file structure:
 
 .. code-block:: text
 
     project/
-      settings.ini
       app/
+        .env
         settings.py
 
-If you call ``config()`` from ``project/app/settings.py`` the library will start looking
+If you call ``config()`` from ``project/app/settings.py`` the library will look
```

Comment:
I'd like to keep the current behavior of looking-up path searching to `.env` environment file by default.

Reply:
Sounds good, will work on that